browser/src/readString.ts

```diff
@@ -0,0 +1,5 @@
+const textDecoder = new TextDecoder();
+
+export default function readString(astBuffer: Uint8Array, start: number, length: number) {
+	return textDecoder.decode(astBuffer.slice(start, start + length));
```

Comment:
Instead of using `.slice()`, which copies part of the ArrayBuffer (see https://devdocs.io/javascript/global_objects/typedarray/slice), you could use `.subarray()`, which should avoid that, see https://devdocs.io/javascript/global_objects/typedarray/subarray.

Reply:
done